packages/common/index.ts

```diff
@@ -96,6 +96,14 @@ export enum Role {
   PROFESSOR = "professor",
 }
 
+/**
```

Comment:
I think we can move this to the eventmodel.entity since it is not shared with the frontend